releasenotes/notes/deprecate-process_config.enabled-89e9ab7b0193cc04.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Added `process_config.process_collection.enabled` and `process_config.container_collection.enabled` settings.
+    Use these new settings instead of `process_config.enabled` to control container and process collection in the process agent.
+  - |
+    Since `process_config.enabled` is deprecated, use `process_config.container_collection.enabled` and
+    `process_config.process_collection.enabled` instead.
+deprecations:
+  - Deprecated `process_config.enabled`.
```

If I understand the feature correctly we have 1 new feature and a deprecation, right ?

```suggestion
features:
  - |
    Configuration `process_config.enabled` is now split into two settings: `process_config.process_collection.enabled` and `process_config.container_collection.enabled`. This will allow better control over the process Agent.
deprecations:
  - Configuration `process_config.enabled` is now deprecated.  Use `process_config.process_collection.enabled` and `process_config.container_collection.enabled` settings instead to control container and process collection in the process Agent.
```

---

Yes, there is 1 new feature and 1 deprecation. Thank you for your suggestion, this changelog makes much more sense